crates/uv-interpreter/src/python_query.rs

```diff
@@ -130,7 +130,7 @@ fn find_python(
         for name in possible_names.iter().flatten() {
             if let Ok(paths) = which::which_in_global(&**name, Some(&path)) {
                 for path in paths {
-                    if cfg!(windows) && windows::is_windows_store_shim(&path) {
+                    if windows::is_windows_store_shim(&path) {
```

Comment:
I don't know how to make code suggestions on ipad…

I would add #[cfg(windows)] to the if block instead of having an empty non-windows implementation in the windows module

Reply:
Sigh, I originally did this but the cross-platform Clippy warnings were so brutal.